packages/charts/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "@hurumap-ui/charts",
+    "version": "0.0.0",
+    "description": "HURUmap charts.",
+    "author": "CodeForAfrica",
+    "license": "GPL-2.0-or-later",
+    "keywords": [
+        "hurumap",
+        "charts"
+    ],
+    "homepage": "https://github.com/CodeForAfrica/HURUmap-UI/tree/master/packages/charts/README.md",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/CodeForAfrica/HURUmap-UI.git",
+        "directory": "packages/charts"
+    },
+    "bugs": {
+        "url": "https://github.com/CodeForAfrica/HURUmap-UI/issues"
+    },
+    "main": "index.js",
+    "devDependencies": {
+        "react": "^16.10.2",
+        "react-dom": "^16.10.2"
+    },
+    "peerDependencies": {
+        "react": "^16.10.2",
+        "react-dom": "^16.10.2"
+    },
+    "dependencies": {
+        "@material-ui/core": "^4.9.3",
+        "apollo-client": "^2.6.8",
+        "apollo-utilities": "^1.3.3",
+        "lodash": "^4.17.15",
+        "prop-types": "^15.7.2",
+        "shortid": "^2.2.15",
+        "victory": "^34.1.1"
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "publish:local": "mkdir -p ../publish-local && yarn pack --filename ../publish-local/$npm_package_name-$npm_package_version.tar && cd ../publish-local && tar xopf $npm_package_name-$npm_package_version.tar --one-top-level=$npm_package_name && cd $npm_package_name && yarn link"
```

Comment:
QQ: Does this work relative to _project_ or _package_ folder?

Reply:
This needs to go sadly.
I will have to figure out how to make a local publish.